api/lib/flight_file_manager/configuration.rb

```diff
@@ -84,6 +85,7 @@ class ConfigError < StandardError; end
       {
         name: 'cloudcmd_cookie_domain',
         env_var: true,
+        default: ->() { RC['flight_WEB_SUITE_domain'] }
```

Comment:
Again, this can be `default: RC['flight_WEB_SUITE_domain']`.

Reply:
0f9b9d430f768fbef5880db88c22e49511ff7be7